document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/AggregationOperator.java

```diff
@@ -6,4 +6,5 @@
   SUM,
   MIN,
   MAX,
+  FIRST,
```

Comment:
will we be able to handle this in `Postgres`?

Reply:
Yup. Just found that this is feasible in Postgres (Ref.: https://stackoverflow.com/a/7630564)